buildstockbatch/kestrel.sh

```diff
@@ -15,7 +15,7 @@ source "$MY_PYTHON_ENV/bin/activate"
 # Default LOCAL_SCRATCH = /tmp/scratch
 # Setting to user-specific dir to avoid
 # issues with deleting previous buildstock run debris
-export LOCAL_SCRATCH=/tmp/scratch/$USER
+export LOCAL_SCRATCH=/tmp/scratch/$SLURM_JOB_ID
```

Comment:
I think you should use $TMPDIR variable as suggested:
`export LOCAL_SCRATCH=$TMPDIR`

Reply:
changed it to `export LOCAL_SCRATCH=$TMPDIR`